### PR TITLE
Alfie698 adds Glasgow Scotland

### LINKED ISSUE #26: alfie698 follows GitHub Flow
# Glasgow, UK

- [x] Create branch
- [x] Add some commits
- [x] Open Pull Request
- [ ] Collaborate
- [ ] Merge 👍 after approval

### PR DESCRIPTION
**Pull request by alfie698**
This request is to add Glasgow to the list of hometowns.

This Pull Request closes #26

### CODE REVIEW

File: alfie698-glasgow.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- Doghouse Merchant City
+
+### Attractions
+
+- Duke Of Wellington
+- Museum Of Transport
+
+### Fun Nearby
+
+- Loch Lomond
```
Comment (Mawer) [inline alfie698-glasgow.md]: What can I do at Loch Lomond?

Comment (alfie698) [inline alfie698-glasgow.md]: Hill walking, wake boarding, water skiing ⛵️ 🏄

**Comment by alfie698** [comment]
@Mawer can you approve this PR? 👍

**Comment by Mawer** [comment]
👍

**Pull request merged**